pkg/operator/ceph/object/controller_test.go

```diff
@@ -754,6 +755,259 @@ func TestCephObjectStoreControllerMultisite(t *testing.T) {
 	})
 }
 
+func TestCephObjectExternalStoreControllerMultisite(t *testing.T) {
```

Comment:
Again As I mentioned before Multisite(Zone/Realm/Zonegroup) configuration are not supported for `ExternalStore`.
Hence I request to check `TestCephObjectStoreController()` than checking `TestCephObjectStoreControllerMultisite()`, last by my mistake I mentioned new function name as ` TestCephObjectExternalStoreControllerMultisite()` it should just be  `TestCephObjectExternalStoreController`

Reply:
Ah, makes sense. I'll rename the function.